core/domain/src/main/java/com/itenum/domain/ContactTelephone.kt

```diff
@@ -0,0 +1,3 @@
+package com.itenum.domain
+
+data class ContactTelephone(var number:String)
```

Comment:
You only need to create one model, no need to create 2 models
You create only one contact model and in this model you add all the information

Reply:
this model correspond a different use case, which is part of a user action derived into an Intent of making a call can not be inside a one single model